src/loglevel-sentry.ts

```diff
@@ -125,7 +138,10 @@ export class LoglevelSentry {
       switch (method) {
         case "error":
           return async (...args: unknown[]) => {
+            // preserve original stack trace as it's lost when using async/await
+            const { stack } = args.find((arg) => arg instanceof Error) || new Error();
             const [err, otherArgs] = await LoglevelSentry.translateError(args);
```

Comment:
when using async/await, we cannot preserve the stack trace when we create a new error. We have to store previous stack and set it later